### PR TITLE
Tournaments: prevent identical tour settings

### PR DESCRIPTION
**Pull request by panpawn**
This helps to avoid resetting various tour settings to what they already are.

### CODE REVIEW

File: tournaments/index.js [hunk 1]
```diff
@@ -910,6 +910,7 @@ let commands = {
 			}
 			if (params[0].toLowerCase() === 'infinity' || params[0] === '0') params[0] = 'off';
```
Comment (TheImmortal) [inline tournaments/index.js]: Can you add the * 60 * 1000 to this line

File: tournaments/index.js [hunk 2]
```diff
@@ -910,6 +910,7 @@ let commands = {
...
 			let timeout = params[0].toLowerCase() === 'off' ? Infinity : params[0];
```
Comment (TheImmortal) [inline tournaments/index.js]: Can you add the ``* 60 * 1000`` to this line

**Comment by panpawn** [comment]
Does that look good?

**Comment by panpawn** [comment]
<b>DO NOT MERGE</b>: I am going to do this for at least one other tour command, so I want to just add it to this PR.  Will do so ASAP.

File: tournaments/index.js [hunk 3]
```diff
@@ -909,8 +910,9 @@ let commands = {
 				}
 			}
 			if (params[0].toLowerCase() === 'infinity' || params[0] === '0') params[0] = 'off';
-			let timeout = params[0].toLowerCase() === 'off' ? Infinity : params[0];
-			if (tournament.setAutoDisqualifyTimeout(timeout * 60 * 1000, this)) {
+			let timeout = params[0].toLowerCase() === 'off' ? Infinity : params[0] * 60 * 1000;
+			if (timeout === tournament.autoDisqualifyTimeout) return this.errorReply("The automatic tournament disqualify timer is already set to this time.");
```
Comment (TheImmortal) [inline tournaments/index.js]: What about using params[0] instead of "this time"

Comment (panpawn) [inline tournaments/index.js]: Done

**Pull request merged**